client/dashboard/top-selling-products/index.js

```diff
@@ -82,7 +82,10 @@ class TopSellingProducts extends Component {
 	}
 
 	render() {
-		const rows = this.getRowsContent( mockData ) || [];
+		const { data, isRequesting, isError } = this.props;
+
+		// TODO: need a loading/empty data indicator
```

Comment:
So I _thought_ we had support in the Table component for an `isLoading` prop but I didn't see one when I just looked right now. Kind of think it would be nice to just add that feature to the table component so we can quickly add a prop and show a loading indicator across all usage. Any thoughts on that @ryelle ?